documentation around AWS SSO integration (#385)
meshcloud/meshcloud-docs: docs/meshstack.aws.index.md

```
<!--END_DOCUSAURUS_CODE_TABS-->

### IAM Configuration

> Currently meshStack supports 2 different ways of integrating AWS IAM with meshStack. The AWS SSO integration is the preferred integration as it

Reviewer: ```suggestion
Currently, meshStack supports 2 different ways of integrating AWS IAM with meshStack, either via AWS SSO or meshIdB (deprecated). The AWS SSO integration is the preferred integration as it
```
As a bonus you could also directly link to these two sections.

> allows using your companies central IdP to log in to AWS. This simplifies integration with meshStack, gives you more control over the AuthN part

Reviewer: ```suggestion
allows using your company's central IdP to log in to AWS. This simplifies integration with meshStack, gives you more control over the AuthN part
```

> and improves UX for endusers when logging in to AWS.

Reviewer: ```suggestion
and improves UX for end-users when logging in to AWS.
```
